examples/extended/grails-app/conf/application.yml

```diff
@@ -193,4 +193,17 @@ grails:
             userLookup:
                 authorityJoinClassName: test.UserRole
                 userDomainClassName: test.User
-            useSwitchUserFilter: true
\ No newline at end of file
+            useSwitchUserFilter: true
+            ui:
```

Comment:
Shouldn't this be organized under `springsecurity`?

Reply:
No, I don't think so based on where the other items are configured in DefaultUiSecurityConfig.groovy in the main plugin.